pkg/Microsoft.Private.CoreFx.NETCoreApp/Microsoft.Private.CoreFx.NETCoreApp.props

```diff
@@ -42,7 +42,6 @@
       <Platform>armel</Platform>
     </OfficialBuildRID>
     <OfficialBuildRID Include="debian.8-x64" />
-    <OfficialBuildRID Include="fedora.23-x64" />
     <OfficialBuildRID Include="fedora.24-x64" />
```

Comment:
Fedora 25 *is* supported, but I'll not do anything about it here. We will certainly see if anything was missing, and I didn't introduce this problem.